### PR TITLE
network docs cleanup

### PR DESCRIPTION
**Pull request by thaJeztah**
This fixes some Markup and formatting issues in the network documentation;
- wrap text to 80 chars
- add missing language hints for code examples
- add missing line continuations ()
- update USAGE output for Cobra

### CODE REVIEW

**Comment by thaJeztah** [comment]
retest this please

File: docs/reference/commandline/network_create.md [hunk 1]
```diff
@@ -10,22 +10,27 @@ parent = "smn_cli"
...
-    --help                   Print usage
-    --internal               Restricts external access to the network
-    --ip-range=[]            Allocate container ip from a sub-range
-    --ipam-driver=default    IP Address Management Driver
-    --ipam-opt=map[]         Set custom IPAM driver specific options
-    --ipv6                   Enable IPv6 networking
-    --label=[]               Set metadata on a network
-    -o --opt=map[]           Set custom driver specific options
-    --subnet=[]              Subnet in CIDR format that represents a network segment
+```markdown
```
Comment (vdemeester) [inline docs/reference/commandline/network_create.md]: markdown ?

**Comment by vdemeester** [comment]
LGTM :+1:

File: docs/reference/commandline/network_create.md (hunk 1 above)
Comment (thaJeztah) [inline docs/reference/commandline/network_create.md]: heh, yes, not entirely accurate, but gave better highlighting 😇 
we should have a "plaintext" hint or something

**Pull request merged**